lib/inquisition/active_record_doctor/runner.rb

```diff
@@ -24,6 +24,8 @@ class Runner < ::Inquisition::Runner
       def call
         TASKS.keys.each do |ard_task|
           ard_task.run.first.each do |table, column|
+            next unless table && column
```

Comment:
As I remember previously ARD return next message `has undefined table references, details: n/a`, what exactly was the problem?

Reply:
@rostikkkk2 ?

Reply:
Needs to rewrite structure TASKS by custom
every ard_task.run returns:
![image](https://user-images.githubusercontent.com/31447852/71482618-4759af80-280c-11ea-980e-fef03a470d9d.png)